.github/workflows/deploy.yml

```diff
@@ -0,0 +1,52 @@
+on:
+  push:
+   tags: v*
+
+env:
+  GITHUB_TOKEN: ${{ secrets.GITHUB_TOKEN }}
+
+jobs:
+  build:
+
+    runs-on: ubuntu-latest
+
+    steps:
+    - name: Check out repo
+      uses: actions/checkout@v2
+
+    - name: Set up JDK 8
+      uses: actions/setup-java@v1
+      with:
+        java-version: 8
+
+    - name: Cache Maven
+      uses: actions/cache@v1
+      with:
+        path: ~/.m2/repository/
+        key: ${{ runner.os }}-maven-${{ hashFiles('**/pom.xml') }}
+        restore-keys: |
+          ${{ runner.os }}-maven-
+
+    - name: Package with Maven
+      id: maven_package
+      run: |
+        mvn package -DskipTests
+        cd ./target/
+        echo "::set-output name=zip_name::$(echo recheck.cli-*-bin.zip)"
```

Comment:
Can't  we use the same wildcard directly in L50?

Reply:
Not that I know … did some research and found this issue:

https://github.com/actions/upload-release-asset/issues/4

Maintainer (@IAmHughes) suggests this approach.